pyproject.toml

```diff
@@ -152,7 +152,7 @@ google-api-core = { version = "^2.15.0", optional = true }
 [tool.poetry.group.dev.dependencies]
 black = "^23.3.0"
 pre-commit = "^3.2.2"
-ruff = "^0.0.220"
+ruff = "^0.1.11"
```

Comment:
Any specific reason to update ruff here? I am not sure if we are ready to update the ruff version yet.

Reply:
`ASYNC` issues were not in v0.0.220.

If the code passes then why not keep the linter current?

Reply:
Thanks for the explanation. 

One more question: What command did you use to update the poetry.lock file?

Reply:
I am no good at Poetry so I tried three things and the fourth one actually did something!  ;-)

I would be happy to revert the changes to `poetry.lock` and then run whatever command you suggest.

Reply:
Yes, can you please remove the changes from the poetry.lock file?

Reply:
`poetry lock --no-update`